tests/installation/check_releasenotes.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check releasenotes during installation
+# Maintainer: mgriessmeier <mgriessmeier@suse.com>, Nick Singer <nsinger@suse.de>
+# Tags: fate#323273, poo#26786
+
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    assert_screen('release-notes-button');
+    send_key('ctrl-shift-alt-x');
```

Comment:
does this work on all backends or only limited ones? What did you use for testing, s390x+VNC or ssh or …? I consider it nicer - and probably more robust - when we use the same approach as for other test modules where we access a terminal from within the installer: `select_console('install-shell')` or any specific reason why we should use an xterm? This also does not work in ncurses, etc.

Reply:
Imo this console is a yast feature which should work in all archs/installation methods.  So if this won't work, I would consider it as a bug

Reply:
*graphical installation methods

Reply:
any specific reason why we should use an xterm?

Reply:
because it's stated in the fate and we would increase our test-coverage by testing this xterm during installation